src/computed-style-property-map.js

```diff
@@ -54,6 +54,29 @@
       case 'z-index':
         return new scope.NumberValue(value);
 
+      // These properties take only a length value (length and percentage),
+      // or a keyword handled above.
+      case 'bottom':
+      case 'height':
+      case 'left':
+      case 'max-height':
+      case 'max-width':
+      case 'min-height':
+      case 'min-width':
+      case 'right':
+      case 'text-indent':
+      case 'top':
+      case 'width':
+        // TODO: support it being any length value.
+        return new scope.SimpleLength(value);
+
+      // These properties only take a length value, but do not take percentage.
```

Comment:
Yep, will add a TODO :-)  Noted that I should add TODO comments for unimplemented things.